Specifying that Python is exclusively one-liners

zdiemer opened the pull request:
It's important to specify here that your Python work is all one-liners.

zdiemer committed:
Specifying that Python is exclusively one-liners

exterkamp commented on README.md:
```diff
@@ -5,7 +5,7 @@ You can usually find me working on...
 * [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report)
 
-These days I write mostly Go, Javascript, and C++; with some Python & Java in the past.
+These days I write mostly Go, Javascript, and C++; with some Python one-liners & Java in the past.
```
Surely this shouldn't be "some" if it's for one-liners?

```suggestion
These days I write mostly Go, Javascript, and C++; with many Python one-liners & Java in the past.
```

zdiemer committed:
Clarify that it is indeed _many_ one-liners

Every interview question we practiced was a one-liner solution, so this quantifies to "many."

Co-authored-by: Shane Exterkamp <shaneexterkamp5@gmail.com>

zdiemer committed:
Reverting removing "some" from referring to Java

exterkamp commented on README.md:
```diff
@@ -5,7 +5,7 @@ You can usually find me working on...
 * [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report)
 
-These days I write mostly Go, Javascript, and C++; with some Python & Java in the past.
+These days I write mostly Go, Javascript, and C++; with many Python one-liners & some Java in the past.
```
```suggestion
These days I write mostly Go, Javascript, and C++; with many Python one-liners & a bit of Java in the past.
```

zdiemer committed:
Using "a bit of" in place of "some"

Using "some" here could be seen as confusing due to the fact that the prior version had "some" referring to both "Python & Java." It would be more clear to simply use "a bit of" here in reference to "Java."

Co-authored-by: Shane Exterkamp <shaneexterkamp5@gmail.com>

zdiemer committed:
Added an extra sentence talking about what Shane does in his free time

Going into some more depth about free-time activities serves to humanize Shane as a character. People viewing his profile will be more likely to emotionally connect with him if we list some interesting free-time activities.

exterkamp commented on README.md:
```diff
@@ -5,7 +5,7 @@ You can usually find me working on...
 * [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report)
 
-These days I write mostly Go, Javascript, and C++; with some Python & Java in the past.
+These days I write mostly Go, Javascript, and C++; with many Python one-liners & a bit of Java in the past. I spend my free time playing Kane & Lynch 2.
```
```suggestion
These days I write mostly Go, Javascript, and C++; with many Python one-liners & a bit of Java in the past.
```

exterkamp reviewed (changes requested):
This is illegal slander.

zdiemer commented:
bump @exterkamp

exterkamp commented:
-1
